com.github.lruzicka.Needly.yaml

```diff
@@ -0,0 +1,113 @@
+# The YAML manifest for the Needle Flatpak
+#
+app-id: com.github.lruzicka.Needly
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: needly
+finish-args: 
+   - --share=ipc # For X11
+   - --socket=wayland # Enable communication with Wayland
+   - --socket=x11 # Alternatively, with X11
+   - --device=kvm # Allow access to a KVM device
+   - --filesystem=home # Allow read-write access to home dir
+   - --filesystem=host # Allow read-write access to host dir tree
```

Comment:
why do you need `host` filesystem? This is very broad. Also it includes `home`

Reply:
Because on the real openQA machine, the "needles" (files that this app edits) could be stored anywhere on the system, most frequently in `/var/lib`. 
On my machine, I maintain two repositories, one for testing and one for development - the majority of openQA users will have the same settings - so I need to be able to open the files from basically everywhere.

Reply:
Maybe if I used `--filesystem=/var/lib/openqa`? Would that enable rw access to that directory and to the subdirectories?

Reply:
The system does not allow writing to `/var/lib` anyway, so I have removed the `host` access.